packages/core/stories/state-icon.doc.stories.mdx

```diff
@@ -0,0 +1,68 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+import { ComponentAnatomy } from "docs/components/ComponentAnatomy";
+import { Button, StateIcon } from "@jpmorganchase/uitk-core";
+import {
+  DownloadIcon,
+  SearchIcon,
+  SendIcon,
+  SettingsSolidIcon,
+} from "@jpmorganchase/uitk-icons";
+import { withFlexGap } from "docs/decorators/withFlexGap";
+import { CustomStylingExample } from "./button.stories";
+import {
+  CSSClassTable,
+  CSSVariableTable,
+  CharacteristicUsage,
+} from "css-variable-docgen-components";
+
+<Meta
+  title="Documentation/Core/StateIcon"
+  component={StateIcon}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# State icon
+
+A state icon can be used on its own or within another component (e.g. `Tooltip`) to help convey a message.
+
+<Canvas>
+  <Story id="core-stateicon--default-state-icon" />
```

Comment:
Purely from documentation perspective, could look nice if we have bigger size

Reply:
Do you mean only on the first icon or all of them?

Reply:
Could be all of the examples. Looks very tiny as a documentation example. @bhoppers2008 to comment :)

Maybe there is ways to set default props in mdx..?

Reply:
Setting default props in mdx doesn't seem to work but I can add them individually to each example. @bhoppers2008 let me know your thoughts :)

Reply:
We can increase the size for documentation but would be best to align it to the work that paul is doing to remove size from the characteristics. That will have some size options. It's WIP though and Paul is on annual leave this week.

Reply:
Makes sense, so should we merge this as is for now and revisit later?